masterlist.yaml

```diff
@@ -2469,10 +2469,12 @@ plugins:
         <<: *dirtyPlugin
         itm: 3
   - name: 'Chesko_Frostfall.esp'
-    req:
-      - name: 'EnchantingAwakenedFrostfallPatch.esp'
-        display: '[Enchanting Awakened Frostfall Patch](http://www.nexusmods.com/skyrim/mods/42796)'
-        condition: 'file("EnchantingAwakened.esp")'
+    msg:
```

Comment:
Please make this a common message in the `common` root list, e.g.

```
- &alsoUseX
  type: say
  content:
    - lang: en
      text: 'It is highly recommended that you also use %1%.'
```

then use that message like:

```
msg:
  - <<: *alsoUseX
    condition: 'file("EnchantingAwakened.esp") and not file("EnchantingAwakenedApocalypsePatch.esp")'
    subs: [ 'the [Enchanting Awakened Apocalypse Patch](http://www.nexusmods.com/skyrim/mods/42796)' ]
```

and similar for the messages you've added below.